src/examples/src/widgets/number-input/LeadingTrailing.tsx

```diff
@@ -11,8 +10,8 @@ export default factory(function Basic() {
 			<NumberInput>
 				{{
 					label: 'Number Input',
-					leading: <Addon>$</Addon>,
-					trailing: <Addon filled>MM</Addon>
+					leading: <span>$</span>,
```

Comment:
I think we should show users that they can pass text here. For most use cases that should be sufficient.
```suggestion
					leading: '$',
```

Reply:
agreed, though I have shown a mix of uses.